cmd/operator/main.go

```diff
@@ -27,18 +27,17 @@ import (
 	"strings"
 	"syscall"
 
-	"github.com/prometheus/client_golang/prometheus"
-	"github.com/prometheus/client_golang/prometheus/promhttp"
-	"golang.org/x/sync/errgroup"
-	"k8s.io/api/core/v1"
-
-	"github.com/coreos/prometheus-operator/pkg/alertmanager"
+	alertmanagercontroller "github.com/coreos/prometheus-operator/pkg/alertmanager"
 	"github.com/coreos/prometheus-operator/pkg/api"
 	monitoringv1 "github.com/coreos/prometheus-operator/pkg/client/monitoring/v1"
 	prometheuscontroller "github.com/coreos/prometheus-operator/pkg/prometheus"
 	"github.com/coreos/prometheus-operator/pkg/version"
 	"github.com/go-kit/kit/log"
 	"github.com/go-kit/kit/log/level"
+	"github.com/prometheus/client_golang/prometheus"
```

Comment:
we did something a little funny here. normally we like to keep imports separated by origin-type:
stdlib, vendored, and project. it should be like it was before but with go-kit placed along with the other vendored dependencies

Reply:
we can clean this up later though